src/app/core/data/signposting-data.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+
+import { catchError, map } from 'rxjs/operators';
+import { Observable, of as observableOf } from 'rxjs';
+
+import { DspaceRestService } from '../dspace-rest/dspace-rest.service';
+import { HALEndpointService } from '../shared/hal-endpoint.service';
+import { RawRestResponse } from '../dspace-rest/raw-rest-response.model';
+import { SignpostingLink } from './signposting-links.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SignpostingDataService {
+
+  constructor(private restService: DspaceRestService, protected halService: HALEndpointService) {
+  }
+
+  getLinks(uuid: string): Observable<SignpostingLink[]> {
```

Comment:
This doesn't appear to be checking if Signposting is enabled.  The backend has a configuration which allows Signposting to be *disabled*.  We should verify that it is enabled before trying to use it.

Reply:
@abollini : I'm ok with this code just handling the 404 response code if signposting is disabled.  I was just pointing out that the REST backend allows for disabling signposting, but the frontend doesn't appear to handle things appropriately if signposting is disabled.

Reply:
@atarix83 : My mistake then... I had thought I tested this and got an error. But, it's possible I neglected to test this :)  In that case, I withdraw this request & I'll double check it's working properly the next time I test this (after other feedback is resolved).